src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const nameContent = `My name is ${person.name}`;
+
+  let partnerContent;
+
+  if (person.isMarried) {
+    partnerContent = person.sex === 'f'
+      ? `${person.partnerName} is my husband`
+      : `${person.partnerName} is my wife`;
+  } else {
+    partnerContent = 'I am not married';
+  }
+
+  let ageContent;
+
+  if (person.age !== undefined) {
+    ageContent = <p className="Person__age">{`I am ${person.age}`}</p>;
+  } else {
+    ageContent = null;
+  }
```

Comment:
here the same, don't create variable for this, just write it in the `return` using `&&` like
```{age && (```

Reply:
Thanks for the feedback! It helped me catch the points I didn't understand before.